### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by ameli90**
1) zoo item pro support a single not repeatable dates
2) zooport update data (categories are not supported)

### CODE REVIEW

File: 02.extensions/zooport/01.basics/03.updating/docs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Update data
+taxonomy:
+    category: docs
+---
+
+It is possible to update element data of existing ZOO items values with CSV. To update items, provide a column with items IDs and set it as **Id** during element assignment step.
```
Comment (miljan-aleksic) [inline 02.extensions/zooport/01.basics/03.updating/docs.md]: Seems the question formating is missing?

Comment (ameli90) [inline 02.extensions/zooport/01.basics/03.updating/docs.md]: it's not F.A.Q. just an article page

File: 02.extensions/zooitempro/01.basics/06.faq/docs.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: F.A.Q.
+taxonomy:
+    category: docs
+---
+
+* **Does ZOOitem Pro filtering support Date Pro element?** <br /> ZOOitem Pro supports single not repeatable date events.
```
Comment (miljan-aleksic) [inline 02.extensions/zooitempro/01.basics/06.faq/docs.md]: change it to "How to filter items based on DatePro element? Currently only single, not repeatable, dates are supported.

File: 02.extensions/zooport/01.basics/03.updating/docs.md (hunk 1 above)
Comment (miljan-aleksic) [inline 02.extensions/zooport/01.basics/03.updating/docs.md]: right...

File: 02.extensions/zooitempro/01.basics/06.faq/docs.md (hunk 2 above)
Comment (ameli90) [inline 02.extensions/zooitempro/01.basics/06.faq/docs.md]: updated

**Pull request merged**